charts/spire/charts/spire-server/templates/tornjak-ingress.tpl

```diff
@@ -0,0 +1,54 @@
+{{- if eq (.Values.tornjak.enabled | toString) "true" }}
```

Comment:
I'm thinking maybe we should skip the ingress stuff entirely for this PR? We can do a full blown tls supported one there.

Reply:
Yes, to simplify things even more, I will remove the ingress for now.